### ISSUE TITLE
Consistent function naming in utils

### ISSUE THREAD

**Issue by mtlam** [nanograv/enterprise#130]
Sometimes you have:
create_ABC versus createABC versus createAB_C

The get_* functions seem more consistently named. Probably best to make them create_ABC to be consistent.

**Comment by jellis18** [comment]
Yes this is a change that needs to be made, we just haven't gotten around to it. Most (if not all) of the `create_` functions are from old PAL2 or NX01 code and the `get_` functions are new.
